assemble/conf/templates/generic_logger.xml

```diff
@@ -25,7 +25,7 @@
      <param name="MaxBackupIndex" value="10"/>
      <param name="Threshold"      value="DEBUG"/>
      <layout class="org.apache.log4j.PatternLayout">
-       <param name="ConversionPattern" value="%d{ISO8601} [%-8c{2}] %-5p: %m%n"/>
+       <param name="ConversionPattern" value="%d{ISO8601} [%-8c{2}] [pid:%6X{pid}] %-5p: %m%n"/>
```

Comment:
A while ago I made local changes todo the following :
 * have the tserver set a pid variable that log4j could use before setting up logging
 * added the pid variable the tservers filename

However this approach was problematic because logs would not be properly rolled and aged off, which could lead to using up lots of disk space for orphaned tserver logs.  This approach of putting the pid in the log file does not have this problem, but it has other problems.  

What guarantees do we have when multiple processes are appending to the same log file?  Are we sure no log messages will be lost?  Also which process make the rollover decision?  If one process rolls over while another is writing to the file could log messages be lost?